src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const total = (cost * days);
+  const largeDiscount = 50;
```

Comment:
The numbers used should have a name for better readability. Consider adding constants like DAILY_COST = 40; for better code readability. 📚